secant/AppDelegate.swift

```diff
@@ -37,9 +37,12 @@ final class AppDelegate: NSObject, UIApplicationDelegate {
         guard !_XCTIsTesting else { return true }
         walletLogger = OSLogger(logLevel: .debug, category: LoggerConstants.walletLogs)
 #endif
-        
         handleBackgroundTask()
 
+        // set the default behavior for the NSDecimalNumber
+        NSDecimalNumber.defaultBehavior = Zatoshi.decimalHandler
+        rootStore.send(.initialization(.appDelegate(.didFinishLaunching)))
```

Comment:
```suggestion

        rootStore.send(.initialization(.appDelegate(.didFinishLaunching)))
```
(since the comment is not related to this line).

Reply:
Updated